storages/python/python-storage-plugins/mongodb.py

```diff
@@ -11,7 +11,7 @@
 
 class mongodb(dlite.DLiteStorageBase):
     """DLite storage plugin for PostgreSQL."""
-    def open(self, uri, options=None):
+    def open(self, uri, options=None, client_options=None):
```

Comment:
The `open()` method should only take `uri` and `options` as arguments. I am afraid we have to encode the client_options in `options`.